### PR TITLE
NEW: Django command to remove inactive users and activation codes

### PR DESCRIPTION
**Pull request by ilhanmercan**

### CODE REVIEW

File: src/accounts/management/commands/clean_inactive_activations_and_accounts.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    activation_code_expiry_time = datetime.datetime.now() - datetime.timedelta(
+        minutes=settings.ACTIVATION_CODE_EXPIRING_MINUTE
+    )
+    expired_activation_code_objects = Activation.objects.filter(created_at__lte=activation_code_expiry_time)
+
+    for activation_code in expired_activation_code_objects:
+        account = activation_code.account
+        if not account.is_active:
+            if not (account.balance > 0 or len(account.domains.all()) > 0 or len(account.payments.all()) > 0):
```
Comment (vesellov) [inline src/accounts/management/commands/clean_inactive_activations_and_accounts.py]: can you put like that : `if account.balance == 0 and len(account.domains.all()) == 0 and account.payments.all() == 0: ` ?

File: src/accounts/management/commands/clean_inactive_activations_and_accounts.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                activation_code.account.delete()
+                logger.warning(f"Inactive account for this email address is removed: '{account.email}'")
+        activation_code.delete()
```
Comment (vesellov) [inline src/accounts/management/commands/clean_inactive_activations_and_accounts.py]: i am not sure, but when you remove account all activation codes related to it suppose to be also removed automatically as collateral ?

Comment (ilhanmercan) [reply src/accounts/management/commands/clean_inactive_activations_and_accounts.py]: that's true. i had to do write 2-3 times write `activation_code.delete()` in if else block. that's why in the end I wrote this. that doesn't do any effect if account is removed.

File: src/accounts/management/commands/clean_inactive_activations_and_accounts.py (hunk 1 above)
Comment (ilhanmercan) [reply src/accounts/management/commands/clean_inactive_activations_and_accounts.py]: changed it

File: src/accounts/management/commands/clean_inactive_activations_and_accounts.py (hunk 2 above)
Comment (ilhanmercan) [reply src/accounts/management/commands/clean_inactive_activations_and_accounts.py]: however, I made it better and more logical now. 
after account deletion, I continue to the other elements of the list.

**Pull request merged**